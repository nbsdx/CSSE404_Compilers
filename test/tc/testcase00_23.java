// bad superclass order

class Main {
    public static void main(String[] args) {
        
    }
}

class Baz extends Foo {
    
    public int baz(boolean abc) {
        return 1;
    }
    
}

class Bar extends Foo {
    int l1;
    int O0;
    
    public int baz(boolean abc) {
        return 0;
    }
    
}

class Foo {
    int foo;
}
