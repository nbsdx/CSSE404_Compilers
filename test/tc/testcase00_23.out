Superclass name Foo not in scope.
Superclass name Foo not in scope.
